apidoc/Titanium/UI/iOS/LivePhotoView.yml

```diff
@@ -60,6 +60,14 @@ events:
         summary: Returns the `playbackStyle` that was provided to start the playback.
         type: Number
 
+properties:
+  - name: livePhoto
```

Comment:
Not sure why this doesn't throw merge conflicts, since the master already contains [this property](https://github.com/appcelerator/titanium_mobile/blob/master/apidoc/Titanium/UI/iOS/LivePhotoView.yml#L22). Can you check? Thanks man!